mypy/build.py

```diff
@@ -1759,9 +1774,13 @@ def parse_file(self) -> None:
                 except (UnicodeDecodeError, DecodeError) as decodeerr:
                     raise CompileError([
                         "mypy: can't decode file '{}': {}".format(self.path, str(decodeerr))])
-            assert source is not None
-            self.tree = manager.parse_file(self.id, self.xpath, source,
-                                           self.ignore_all or self.options.ignore_errors)
+
+            if not self.namespace:
```

Comment:
A small nit:
```
if <condition>:
   action
else:
   other action
```
rather than
```
if not <condition>:
    other action
else:
    action
```